fuse/data/ops/ops_visprobe.py

```diff
@@ -26,44 +70,50 @@ class VisProbe(OpBase):
 
     Important notes:
     - running in a cached environment is dangerous and is prohibited
-    - this Operation is not thread safe ans so multithreading is also discouraged
+    - this Operation is not thread safe and so multithreading is also discouraged
 
     "
     """
 
     def __init__(self,flags: VisFlag, 
                  keys: Union[List, dict] ,
                  type_detector: TypeDetectorBase,
-                 id_filter: Union[None, List] = None, 
+                 coco_converter : Callable = None,
```

Comment:
coco is too specific,  we might want to use different converters. You can just call it convertor

Reply:
what I mean is that a user must use COCO format for segmentations or have a function that converts to COCO format.
that's what we talked about before.. 
we don't want to support more than 1 format for each seg type

Reply:
It's maybe true for imaging, but about the other domains?
Or if we decide to support more options in the future?

Reply:
coco converter moved to imaging visualizer